.circleci/config.yml

```diff
@@ -213,6 +213,7 @@ jobs:
       - persist_to_workspace:
           root: ~/react-widgets
           paths:
+            - package.json
```

Comment:
Persisting the package.json file that has been updated with the latest version. Grabbing the version in the `deploy_to_cdn` was reporting the previous version.